src/PackageCompiler.jl

```diff
@@ -1599,6 +1607,25 @@ function bundle_headers(dest_dir, header_files)
     return
 end
 
+function audit_sysimage_relocatability(sysimg_path::String; paths::Vector{String} = [homedir(), DEPOT_PATH...])
+
+    none_found = true
+    sysimg_contents = open(io -> String(read(io)), sysimg_path, read=true)
+
+    for path in paths
+        found = String[]
+        for m in eachmatch(Regex("$(path)[^\0]+"), sysimg_contents)
```

Comment:
Looks like this regex causes problems on Windows because of the backslashes in Windows paths.

```suggestion
        for m in eachmatch(Regex("\Q$(path)\E[^\0]+"), sysimg_contents)
```